servatrice/servatrice.ini.example

```diff
@@ -99,6 +99,15 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Enable forgot password functionality
+enableForgotPassword=true
+
+; Validate users inputed email matches email on account before allowing forgot password reset
+forgotpasswordemailrequired=true
+
+; Validate users clientid matches clientid on account before allowing forgot password reset
+forgotpasswordclientidrequired=true
```

Comment:
when enabled, servatrice will check if the clientid of the cockatrice instance requesting the password reset matches the last-known clientid of the user the password reset was requested for.